Mathlib/Order/WithBot.lean

```diff
@@ -338,16 +338,18 @@ instance preorder [Preorder α] : Preorder (WithBot α) where
     let ⟨c, hc, bc⟩ := h₂ b hb
     ⟨c, hc, le_trans ab bc⟩
 
-instance partialOrder [PartialOrder α] : PartialOrder (WithBot α) :=
-  { WithBot.preorder with
```

Comment:
For simplicity, best not to make random change like the `__` change here. (Potentially subtle consequences, and orthogonal to everything else, so make an easy to review PR not quite as easy.)